client/src/main/java/io/pravega/client/stream/impl/Controller.java

```diff
@@ -249,6 +249,15 @@
      */
     CompletableFuture<Set<Segment>> getSuccessors(StreamCut from);
 
+    /**
+     * Get Segments Inclusive between two streamCuts... TODO update the documentation.
+     *
+     * @param fromStreamCut FromStreamCut...
+     * @param toStreamCut ToStreamCut...
+     * @return list of segments ...
+     */
+    CompletableFuture<Set<Segment>> getSegmentsInclusive(StreamCut fromStreamCut, StreamCut toStreamCut);
```

Comment:
nit: the method name is odd.
`getSegments` should be good name

Reply:
done.